src/infinite_sets.jl

```diff
@@ -397,6 +397,14 @@ supports. This is done by invoking [`generate_unique_label`](@ref).
 """
 struct UniqueMeasure{S} <: PublicLabel end
 
+"""
+    MeasureBound <: PublicLabel
+    
+A support label for supports that are generated using the upper and lower bounds
+for `FunctionalDiscreteMeasureData`.
+"""
+struct MeasureBound <: PublicLabel end
+
```

Comment:
This docstring should be added to the docs with the other support label docstrings